tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/Debugger/snapshots/ProbesTests.AsyncGenericClass.verified.txt

```diff
@@ -69,7 +69,7 @@
           "id": "ScrubbedValue",
           "location": {
             "method": "Method",
-            "type": "Samples.Probes.SmokeTests.AsyncGenericClass+NestedAsyncGenericClass`1[[Samples.Probes.Shared.Generic, Samples.Probes, Version=1.0.0.0, Culture=neutral, PublicKeyToken=def86d061d0d2eeb]]"
+            "type": "Samples.Probes.Contracts.SmokeTests.AsyncGenericClass+NestedAsyncGenericClass`1[[Samples.Probes.Contracts.Shared.Generic, Samples.Probes.Unoptimized, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null]]"
```

Comment:
Question: Do you expect the same snapshots from both the Optimized and Unoptimized samples?

- If yes, then I think you should scrub the `.Unoptimized` from this snapshots, so that they share correctly. 
- If no, you should probably add `.Unoptimized` to the snapshot path


I suspect you may want to scrub the `Unobtimized